websites/A/AnimesOnline.Games/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "AnimesOnline.Games",
```

Comment:
If they changed url, I would name it now
`AnimesOnlineGames` without dot.

Reply:
Yeah, that seems like the reasonable thing to do